spec/breadcrumbs_spec.rb

```diff
@@ -115,9 +115,10 @@ def breadcrumb_links
   end
 
   def page
-    path = Faker::Internet.url
+    url = Faker::Internet.url # "https://example.com/lorem_ipsum"
+    path = URI(url).path[1..-1] # "lorem_ipsum"
```

Comment:
I assume you're trying to remove a leading slash. If so, how about this?
```suggestion
    path = URI(url).path.gsub %r{^/}, ''
```